book.rb

```diff
@@ -1,8 +1,14 @@
 class Book
-  attr_accessor :title, :author
+  attr_accessor :title, :author, :rentals
 
   def initialize(title, author)
     @title = title
     @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
```

Comment:
- Hey, great job adding this method but, we still need to add a date parameter to keep track of the date the book is being rented and the person itself is sent to the parameter. See below:

```ruby
 def add_rental(person, date)
    Rental.new(date, person, self)
  end
``